components/widgets/icons/fa_next_icon.tsx

```diff
@@ -25,7 +24,7 @@ export default class NextIcon extends React.PureComponent {
                 {(title) => (
```

Comment:
Shouldn't this be:
```suggestion
                {(title: string) => (
```

That should fix your issue below.

Reply:
I tried it before. I get an error from the type check. 
```
Type '(title: string) => Element' is not assignable to type '(...formattedMessage: (string | Element)[]) => ReactNode'.
    Types of parameters 'title' and 'formattedMessage' are incompatible.
      Type 'string | Element' is not assignable to type 'string'.
        Type 'Element' is not assignable to type 'string'.
```

Reply:
Maybe one way to get away is to use the `formatMessage` API instead of `FormattedMessage` component.

Reply:
What do you think about this version:
```
{(title): JSX.Element => {
                    if (typeof title === 'string') {
                        return (<i
                            className={className}
                            title={title}
                        />)
                    }
                    return (<i/>)
                }}
```
It passes the tests. I'm just not sure what to do in the case if title was an Element instead of a string. I'm not sure that it ever happens though.

Reply:
```
{(...elemArray: (string | JSX.Element)[]): JSX.Element => {
                    const [title] = elemArray
                    if (typeof title === 'string') {
                        return (<i
                            className={className}
                            title={title}
                        />)
                    }
                    return (<i/>)
                }}
```
This version might be a little bit better. I still don't know what to do for the case when title is an element.

Reply:
My proposal is something like:
```javascript
    public render(): JSX.Element {
        const {formatMessage} = this.context.intl;
        const className = 'fa fa-1x fa-angle-right' + (this.props.additionalClassName ? ' ' + this.props.additionalClassName : '');
        return (
            <i
                className={className}
                title={formatMessage({id: 'generic_icons.next', defaultMessage: 'Next Icon'})}
            />
        );
    }
```